source/admin_manual/logging.rst

```diff
@@ -139,9 +139,9 @@ There are several settings that control logging verbosity. By default they're al
 
 * :dovecot_core:ref:`auth_verbose=yes <auth_verbose>` enables logging all failed authentication attempts.
 
-* :dovecot_core:ref:`auth_debug=yes <auth_debug>` enables all authentication debug logging (also enables :dovecot_core:ref:`auth_verbose`). Passwords are logged as `<hidden>`.
+* :dovecot_core:ref:`log_debug = category=auth <log_debug>` enables all authentication debug logging (also enables :dovecot_core:ref:`auth_verbose`). Passwords are logged as `<hidden>`.
```

Comment:
I wonder if this is true anymore. We should check and add CI test maybe.

Reply:
I get e.g.: Jul 05 10:40:54.396001 auth: Info: passdb(tss,127.0.0.1,<fkny+AnjJNx/AAAB>): Password mismatch